### PR TITLE
Training scheduler pages integrated

### PR DESCRIPTION
**Pull request by ShahinaKP**
Hi @nishintn 

Please review the branch for following implementations and merge with master.

- Menu bar integrated with routing
- Created a custom component for listing trainings in "Trainings"  and "My Trainings" pages

### CODE REVIEW

File: app/trainingScheduler/mytrainings/mytrainings.component.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import { Component, Input, OnInit} from "@angular/core";
+import { RouterModule, Router } from "@angular/router";
+
+import { TrainingsMainService } from "../trainings/trainingsmain.service";
+import { TrainingListingComponent } from "../sharedcomponent/trainingListComp.component";
+import { LoginComponent } from "../../login/login.component";
+
+@Component({
+  selector: "my-trainings",
```
Comment (mad-nishin) [inline app/trainingScheduler/mytrainings/mytrainings.component.ts:9]: my-trainings??

File: app/trainingScheduler/mytrainings/mytrainings.component.ts [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  templateUrl: "app/trainingScheduler/mytrainings/mytrainings.component.html",
+  styleUrls:  ["app/trainingScheduler/trainings.component.css"]
+})
+
+
+export class MyTrainingsComponent implements OnInit {
```
Comment (mad-nishin) [inline app/trainingScheduler/mytrainings/mytrainings.component.ts:15]: MyTrainingsComponent??

File: app/trainingScheduler/mytrainings/mytrainings.component.ts (hunk 1 above)
Comment (ShahinaKP) [reply app/trainingScheduler/mytrainings/mytrainings.component.ts:9]: This component is for listing the user's trainings and so component name is given as "my-trainings"

File: app/trainingScheduler/mytrainings/mytrainings.component.ts (hunk 2 above)
Comment (ShahinaKP) [reply app/trainingScheduler/mytrainings/mytrainings.component.ts:15]: This component is for listing the user's trainings and so component name is given as MyTrainingsComponent

**Pull request merged**